src/main/java/uk/gov/hmcts/divorce/citizen/notification/conditionalorder/AppliedForConditionalOrderNotification.java

```diff
@@ -41,7 +41,7 @@
     static final String APPLICANT1 = "applicant 1";
     static final String APPLICANT2 = "applicant 2";
 
-    private String submittingUserId;
+    private String submittingUser;
```

Comment:
This is not your doing but we should not have local state here. This class will live beyond the lifetime of a request and if someone forgets to set this variable before calling them method to sendToApplicantX it will do the wrong this. And it's just bad design.

Please make the sendToApplicantX methods stateless and remove this property

Reply:
done